File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webapi/service/UaServerConfigure.java
package org.opcfoundation.webapi.service;

import org.eclipse.milo.opcua.stack.core.types.builtin.LocalizedText;
import org.eclipse.milo.opcua.stack.core.types.enumerated.ApplicationType;

public class UaServerConfigure {
    private String applicationUri;
    private LocalizedText applicationName;
    private String productUri;
    private ApplicationType applicationType;

    private boolean supportServerUriPath;

    private int browseRequestMaxSize;
    private int readRequestMaxSize;
    private int writeRequestMaxSize;
    private int callRequestMaxSize;
    private int historyReadRequestMaxSize;
    private int translateRequestMaxSize;

    public UaServerConfigure()
    {
        applicationUri = "Unknown";
        applicationName = new LocalizedText("Unknown");
        productUri = "Unknown";
        applicationType = ApplicationType.Server;

        supportServerUriPath = true;

        browseRequestMaxSize = 0;
        readRequestMaxSize = 0;
        writeRequestMaxSize = 0;
        callRequestMaxSize = 0;
        historyReadRequestMaxSize = 0;
        translateRequestMaxSize = 0;
    }

    public ApplicationType getApplicationType() {
        return applicationType;
    }

    public void setApplicationType(ApplicationType applicationType) {
        this.applicationType = applicationType;
    }

    public String getApplicationUri() {
        return applicationUri;
    }

    public void setApplicationUri(String applicationUri)
    {
        this.applicationUri = applicationUri;
    }

    public LocalizedText getApplicationName() {
        return applicationName;
    }

    public void setApplicationName(LocalizedText applicationName)
    {
        this.applicationName = applicationName;
    }

    public String getProductUri()
    {
        return productUri;
    }

    public void setProductUri(String productUri)
    {
        this.productUri = productUri;
    }

    public boolean isServerUriPathSupported() {
        return supportServerUriPath;
    }

    public void setSupportServerUriPath(boolean isSupported) {
        supportServerUriPath = isSupported;
    }

    public int getBrowseRequestMaxSize() {
        return browseRequestMaxSize;
    }

    public void setBrowseRequestMaxSize(int browseRequestMaxSize) {
        this.browseRequestMaxSize = Math.max(browseRequestMaxSize, 0);;
    }

    public int getReadRequestMaxSize() {
        return readRequestMaxSize;
    }

    public void setReadRequestMaxSize(int readRequestMaxSize) {
        this.readRequestMaxSize = Math.max(readRequestMaxSize, 0);
    }

    public int getWriteRequestMaxSize() {
        return writeRequestMaxSize;
    }

    public void setWriteRequestMaxSize(int writeRequestMaxSize) {
        this.writeRequestMaxSize = Math.max(writeRequestMaxSize, 0);
    }

    public int getCallRequestMaxSize() {
        return callRequestMaxSize;
    }

    public void setCallRequestMaxSize(int callRequestMaxSize) {
        this.callRequestMaxSize = Math.max(callRequestMaxSize, 0);
    }

    public int getHistoryReadRequestMaxSize() {
        return historyReadRequestMaxSize;
    }

    public void setHistoryReadRequestMaxSize(int historyReadRequestMaxSize) {
        this.historyReadRequestMaxSize = Math.max(historyReadRequestMaxSize,0);
    }

    public int getTranslateRequestMaxSize() {
        return translateRequestMaxSize;
    }

    public void setTranslateRequestMaxSize(int translateRequestMaxSize) {
        this.translateRequestMaxSize = Math.max(translateRequestMaxSize, 0);
    }
}

File: opcua-webservice-node/src/server/types/common/UaServerConfigure.ts
import { ApplicationType } from "opcua-webapi";
import { UaLocalizedText } from "opcua-webapi-ts";

export class UaServerConfigure {
    private _applicationUri : string;
    private _applicationName : UaLocalizedText;
    private _productUri : string;
    private _applicationType : ApplicationType;

    private _supportServerUriPath : boolean;

    private _browseRequestMaxSize : number;
    private _readRequestMaxSize : number;
    private _writeRequestMaxSize : number;
    private _callRequestMaxSize : number;
    private _historyReadRequestMaxSize : number;
    private _translateRequestMaxSize : number;

    constructor()
    {
        this._applicationUri = "Unknown";
        this._applicationName = new UaLocalizedText("Unknown");
        this._productUri = "Unknown";
        this._applicationType = ApplicationType.Server;

        this._supportServerUriPath = false;

        this._browseRequestMaxSize = 0;
        this._readRequestMaxSize = 0;
        this._writeRequestMaxSize = 0;
        this._callRequestMaxSize = 0;
        this._historyReadRequestMaxSize = 0;
        this._translateRequestMaxSize = 0;
    }

    get applicationType() : ApplicationType 
    {
        return this._applicationType;
    }

    set applicationType(applicationType : ApplicationType) {
        this._applicationType = applicationType;
    }

    get applicationUri() : string {
        return this._applicationUri;
    }

    set applicationUri(applicationUri: string)
    {
        this._applicationUri = applicationUri;
    }

    get applicationName() : UaLocalizedText 
    {
        return this._applicationName;
    }

    set applicationName(applicationName : UaLocalizedText)
    {
        this._applicationName = applicationName;
    }

    get productUri() : string
    {
        return this._productUri;
    }

    set productUri(productUri : string)
    {
        this._productUri = productUri;
    }

    get supportServerUriPath() : boolean
    {
        return this._supportServerUriPath;
    }

    set supportServerUriPath(isSupported : boolean) 
    {
        this._supportServerUriPath = isSupported;
    }

    get browseRequestMaxSize() : number 
    {
        return this._browseRequestMaxSize;
    }

    set browseRequestMaxSize(browseRequestMaxSize : number) 
    {
        this._browseRequestMaxSize = Math.max(browseRequestMaxSize, 0);
    }

    get readRequestMaxSize() : number
    {
        return this._readRequestMaxSize;
    }

    set readRequestMaxSize(readRequestMaxSize : number) {
        this._readRequestMaxSize = Math.max(readRequestMaxSize, 0);
    }

    get writeRequestMaxSize() : number{
        return this._writeRequestMaxSize;
    }

    set writeRequestMaxSize(writeRequestMaxSize : number) {
        this._writeRequestMaxSize = Math.max(writeRequestMaxSize, 0);
    }

    get callRequestMaxSize() {
        return this._callRequestMaxSize;
    }

    set callRequestMaxSize(callRequestMaxSize : number) {
        this._callRequestMaxSize = Math.max(callRequestMaxSize, 0);
    }

    get historyReadRequestMaxSize() {
        return this._historyReadRequestMaxSize;
    }

    set historyReadRequestMaxSize(historyReadRequestMaxSize: number) {
        this._historyReadRequestMaxSize = historyReadRequestMaxSize;
    }

    get translateRequestMaxSize() {
        return this._translateRequestMaxSize;
    }

    set translateRequestMaxSize(translateRequestMaxSize : number) {
        this._translateRequestMaxSize = Math.max(translateRequestMaxSize, 0);
    }
}